infrastructure/main.tf

```diff
@@ -32,3 +32,18 @@ data "azurerm_key_vault" "key_vault" {
   name                = "${module.vault.key_vault_name}"
   resource_group_name = "${azurerm_resource_group.rg.name}"
 }
+
+resource "azurerm_application_insights" "appinsights" {
+  name                = "${var.product}-${var.env}"
+  location            = "${var.location}"
+  resource_group_name = "${azurerm_resource_group.rg.name}"
+  application_type    = "${var.application_type}"
+  tags                = "${var.common_tags}"
+}
+
+# store app insights key in key vault
+resource "azurerm_key_vault_secret" "bulk_scan_orchestrator_app_s2s_secret" {
```

Comment:
name should be app insights and not secret ?

Reply:
🙊 . let's agree together - you didn't see it :D